universal-login-react/src/ui/TopUp/Fiat/TopUpWithFiat.tsx

```diff
@@ -57,7 +57,8 @@ export const TopUpWithFiat = ({sdk, onPayClick, logoColor}: TopUpWithFiatProps)
   useAsyncEffect(recognizeUserCountry, []);
 
   const isPayButtonDisabled = !paymentMethod ||
-    (topUpProviderSupportService.isInputAmountUsed(paymentMethod) && !amount);
+    (topUpProviderSupportService.isInputAmountUsed(paymentMethod) && !amount) ||
+    paymentMethod === TopUpProvider.WYRE;
```

Comment:
Seems like it should be in a separate PR?

Reply:
Done